genesis-tools.Jenkinsfiles

```diff
@@ -0,0 +1,40 @@
+// Params in JobDSL file
+// 'https://gitlab.com/Concordium/infra/jenkins-jobs/-/blob/master/docker_image_genesis_tools.groovy':
+// - image_tag (default: "latest")
+// - base_image_tag
+
+pipeline {
+    agent any
+    environment {
+        image_repo = 'concordium/genesis-tools'
+	image_name = "${image_repo}:${image_tag}"
+    }
+    stages {
+	stage('dockerhub-login') {
+            environment {
+                CRED = credentials('jenkins-dockerhub')
+            }
+            steps {
+                sh 'echo $CRED_PSW | docker login --username $CRED_USR --password-stdin'
+            }
+        }
+        stage('build') {
+            steps {
+                sh '''\
+                    docker build \
+                        --build-arg BASE_IMAGE_TAG="$base_image_tag" \
```

Comment:
Doesn't this mean you'd have to update the `genesis-tools.Dockerfile` by parametrizing it with this image?